go.mod

```diff
@@ -1,55 +1,59 @@
 module k8s.io/kube-state-metrics/v2
 
+go 1.19
+
 require (
 	github.com/dgryski/go-jump v0.0.0-20211018200510-ba001c3ffce0
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/gobuffalo/flect v1.0.2
 	github.com/google/go-cmp v0.5.9
 	github.com/oklog/run v1.1.0
-	github.com/prometheus/client_golang v1.14.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus v0.75.0
+	github.com/prometheus/client_golang v1.15.0
 	github.com/prometheus/client_model v0.3.0
 	github.com/prometheus/common v0.42.0
 	github.com/prometheus/exporter-toolkit v0.9.1
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/spf13/cobra v1.6.1
+	github.com/spf13/cobra v1.7.0
 	github.com/spf13/viper v1.15.0
 	github.com/stretchr/testify v1.8.2
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.2
-	k8s.io/apimachinery v0.26.2
-	k8s.io/client-go v0.26.2
-	k8s.io/component-base v0.26.2
+	k8s.io/api v0.27.1
```

Comment:
do you need to upgrade those for specific dependencies? Could we stay with 0.26.x and move to 0.27.x after 2.9.0 is released?